contribute.md

```diff
@@ -1,7 +1,60 @@
+## Contributing
+
+Hey there! We're thrilled that you'd like to contribute to this project. Your help is essential for keeping it great.
+
+Please note that this project is released with a [Contributor Code of Conduct](https://github.com/srinidh-007/HackotberFest2021). By participating in this project you agree to abide by its terms.
+
 # Rules & Regulations 
 
 -   This is Beginner Friendly, Kickstart for students and those who are new to opensource 
 -   Use any language.
 -   Don't spam it will be rejected immediately
 -   Python, C, C++, Java, HTML, CSS, JavaScript, Node, Data Structure and Algorithms, Android Projects.
 -   Anything valuable for Community
+
+## Issues and PRs
+
+This is a beginner friendly repository where all those people who are new to open source can contribute. The unique thing is you can use any
+language of your choice be it Python, C, C++, Java, HTML, CSS, JavaScript, Node, Data Structure and Algorithms, Android Projects which would be considered valuable to the open source community as a whole. Interested? See the #how to contribute.
+
+## How to Contribute
+
+- Take a look at the Existing [Issues](https://github.com/srinidh-007/HackotberFest2021.git) or create your own Prs with any language!
+- Fork the Repo so that you have a local copy then clone it into your PC.
+- Create a Pull Request which will be promptly reviewed and suggestions would be added to improve it.
+- Add Screenshots to help us know what this is all about.
+
+## How to make a Pull Request
+
+**1.** Fork the repository by clicking on the <a href="https://github.com/srinidh-007/HackotberFest2021"><img src="https://img.icons8.com/ios/24/000000/code-fork.png"></a> symbol at the top right corner.
+
+**2.** Clone the forked repository.
+```
+   git clone https://github.com/srinidh-007/HackotberFest2021.git
+```
+
+**3.** Make changes in source code using any code editor.
+
+**4.** Stage your changes and commit
+
+```
+   git add .
+```
+
+```
+   git commit -m "<your_commit_message>"
+```
+
+**5.** Push your local commits to the remote repo.
+
+```
+   git push 
```

Comment:
in git push, if the upstream path is not specified at first it will complain unless you already added the path in your local directory